yarn.lock

```diff
@@ -3381,6 +3382,11 @@
   resolved "https://registry.yarnpkg.com/@sentry/core/-/core-8.48.0.tgz#3bb8d06305f0ec7c873453844687deafdeab168b"
   integrity sha512-VGwYgTfLpvJ5LRO5A+qWo1gpo6SfqaGXL9TOzVgBucAdpzbrYHpZ87sEarDVq/4275uk1b0S293/mfsskFczyw==
 
+"@sentry/core@^8.42.0":
```

Comment:
duplicate is kinda interesting. might get remove with `npx yarn-deduplicate && yarn install`

Reply:
will do!

Reply:
hrrm ran this, and didn't get any changes to occur 🤔 

We will be in the near future hopefully bundling all deps inside of the plugins so this should disappear once we get that all sorted out.